metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/MetadataImporterMainApplication.java

```diff
@@ -97,9 +97,8 @@ public void run(ApplicationArguments applicationArguments) throws Exception {
         String accessionFilePath = accessionsFilePath.get(0);
         Set<String> accessions;
         try {
-            accessions = new HashSet<>(Files.readAllLines(Paths.get(MetadataImporterMainApplication.class
-                    .getClassLoader().getResource(accessionFilePath).toURI())));
-        } catch (NullPointerException | URISyntaxException exception) {
+            accessions = new HashSet<>(Files.readAllLines(Paths.get(accessionFilePath)));
```

Comment:
so, the rationale to simplify the code here is because we will always be using local files?

Reply:
Exactly. Indeed, I don't know what was the rationale to define this as a resource file in the first place. The list of studies is always a simple local file supplied prior to running, not in any way part of the package